packages/client/src/hooks/useFlowFees.js

```diff
@@ -0,0 +1,24 @@
+import { query } from "@onflow/fcl";
+import { GET_FLOW_FEES_ESTIMATION } from "../flow";
+import {
+  EXECUTION_EFFORTS,
+  DEFAULT_INCLUSION_EFFORT,
+} from "constants/constantValues";
+
+const useFlowFees = () => {
+  const getEstimation = async () => {
+    const { PROPOSE_ACTION, SIGN_ACTION, EXECUTE_ACTION } = EXECUTION_EFFORTS;
+    const executionEffort = PROPOSE_ACTION + SIGN_ACTION + EXECUTE_ACTION;
```

Comment:
I'm not sure if we should just sum these three up, because we are having 3 separate actions here, and each should have its own estimation?

Reply:
base on the design, I made the assumption that it's a sum, watching the [conversation here](https://brud.slack.com/archives/C03ERT4F2MB/p1661187075427139) for the direction

Reply:
Yeah, I don't think it should be summed.  Instead, `getEstimation` should accept tx type as an argument, and `executionEffort` should be assigned based on that.